### DISCUSSION TITLE
可以像TableStructureRec一样输出一个logic_point吗

### DISCUSSION THREAD

**Discussion by enymfc25173** [RapidAI/RapidTable#23] [Ideas]
如题

**Comment by Joker1212** [comment]
理论上也可以，就是得改造一下，因为它的输出实际上是<td><th>这样的标签，后面有时间了我改一下吧

**Comment by enymfc25173** [comment]
顺便请教下RapidTable的微调及onnx转换有参考的文档吗

**Reply by Joker1212** [reply]
微调的话，直接参考 paddleX 上的做法，转换onnx也是标准的使用paddle2onnx工具就行了，注意训练的时候不要改字典，理论上都是<td>这种标签，其实也不需要改动字典的https://paddlepaddle.github.io/PaddleX/latest/pipeline_usage/tutorials/ocr_pipelines/table_recognition.html

**Comment by Joker1212** [comment]
0.3.0版本已经支持返回和TableStructureRec相同的逻辑坐标格式以及支持可视化了
